templates/cloud-native-kubernetes-usage-report-2021/index.html

```diff
@@ -5113,8 +5113,8 @@ <h4>What are the TWO most important questions that ops people should care about?
 
       <p>To execute on these scenarios, we believe that software should be running your application.</p>
 
-      <p>The Charmed Operator Framework (also called Juju) brings business thinking to the world of application management, making it much easier to have conversations between teams about the estate, and much easier to evolve capacity allocation over time: you can easily manage multiple compute environments and lower your costs of ownership. The separation of technical and commercial concerns means the operator is reusable in very different business settings. And reuse enhances the community value of the operator and increases software quality.</p>
-      <p>Most importantly, the Charmed Operator Framework is not a &ldquo;language for configuration file management&rdquo;. It is not a successor to Puppet, Chef, Ansible, Terraform or Salt. Juju removes the requirement to invest in deep institutional knowledge of configuration details altogether, by encapsulating community knowledge in operators that handle those details automatically, given a context. Moving beyond configuration management puts the focus on application management and resource allocation. It is the stepping stone to Model Driven Operations.</p>
+      <p>The Juju brings business thinking to the world of application management, making it much easier to have conversations between teams about the estate, and much easier to evolve capacity allocation over time: you can easily manage multiple compute environments and lower your costs of ownership. The separation of technical and commercial concerns means the operator is reusable in very different business settings. And reuse enhances the community value of the operator and increases software quality.</p>
```

Comment:
Ah THE Juju. I think I saw a post on The Facebook about it.

Reply:
:rofl: